### PR TITLE
{debugger}[gompi/2019b] Valgrind v3.16.1

### PR DESCRIPTION
**Pull request by tstrempel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/v/Valgrind/Valgrind-3.16.1-gompi-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'Valgrind'
+version = '3.16.1'
+
+homepage = 'https://valgrind.org'
+description = "Valgrind: Debugging and profiling tools"
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'optarch': False}
```
Comment (smoors) [inline easybuild/easyconfigs/v/Valgrind/Valgrind-3.16.1-gompi-2019b.eb:10]: @tstrempel why setting optarch to False? I tested it without this option and it builds fine.

Comment (tstrempel) [reply easybuild/easyconfigs/v/Valgrind/Valgrind-3.16.1-gompi-2019b.eb]: @smoors I took this option from an older EasyConfig. As it builds without it, I'll remove it.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node361.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/d117bdaf772002d9b791c6ca791b6ccd for a full test report.

**Comment by smoors** [comment]
Going in, thanks @tstrempel!

**Pull request merged**